docs/docs/Contributing/contributing-community.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Community
-sidebar_position: 3
+sidebar_position: 5
 slug: /contributing-community
 ---
 
```

Comment:
I'm unable to comment on the remaining text so I'm adding it here. 

- Remove the icons from the headings. 
- Change "Join Langflow Discord server" to "Join the Langflow Discord server". Remove the bold from Langflow.
- Replace "Join us to ask questions and showcase your projects." with "Join the [Langflow Discord Server](https://discord.gg/EqksyE2EX9) to ask questions and showcase your projects."
- Remove "Let's bring together the building blocks of AI integration!"
- Remove "Langflow [Discord](https://discord.gg/EqksyE2EX9) server"
- Change "Stay tuned for Langflow on Twitter" to "Follow Langflow on X". 
- In "Follow [@langflow_ai](https://twitter.com/langflow_ai) on **Twitter** to get the latest news about **Langflow**.", verify the URL and update Twitter to X. Also, there's no reason to put Langflow in bold. 
- Change "You can "watch" **Langflow** in [GitHub](https://github.com/langflow-ai/langflow). If you select "Watching" instead of "Releases only" you will receive notifications when someone creates a new issue or question. You can also specify that you only want to be notified about new issues, discussions, PRs, etc. so you can try and help them solve those questions." to "You can [watch Langflow in GitHub](https://github.com/langflow-ai/langflow). If you select **Watching** instead of **Releases only** you will receive notifications when someone creates a new issue or question. You can also specify that you want to be notified only about new issues, discussions, and PRs so you can try to help solve those issues."

Reply:
Thank you